recipes/molearn/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "molearn" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ed43bbe23ec5be2cd25a8e5970fa255f67404ec62bdb269ba3489d97be3bcb8b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win ]
```

Comment:
It seems like the package can be noarch. If the only reason this is not is the missing dependency, that error can be ignored and it should still be noarch.

Reply:
This recipe will never be available to run on windows, our planned future versions will be even less able to run on windows.

Reply:
In that case, please still make it noarch since I cannot see anything else that prevents it and add a virtual package (__unix) to the run section: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html